.gitignore

```diff
@@ -73,3 +73,12 @@ id_rsa_*
 
 # python things
 __pycache__
+
```

Comment:
Could you remove this change? 

You can actually add a .gitignore file on these folders, and have it ignore just those files.

Reply:
moved to my user folder